test/integration/targets/dnf/tasks/repo.yml

```diff
@@ -560,42 +535,43 @@
   always:
     - name: Clean up
       dnf:
-        name: dinginessentail
+        name: multilib-dinginessentail
         state: absent
+  when: ansible_architecture == "x86_64"
```

Comment:
Can this portion of the test be adapted so it will also work when run using aarch64?

You can check aarch64 behavior by running `ansible-test` with the `--remote-arch aarch64` option.